### ISSUE TITLE
Autocrafting with AE2 and some Mekanism machines not working

### ISSUE THREAD

**Issue by Trostan** [AllTheMods/ATM-3#515]
I can't autocraft Chemical Dissolution Chamber, Chemical Washer or Chemical Crystallizer

When trying to craft one of these machines it just crafts all needed parts and they 
all say: "Stored: (number)" and the main item i want to craft says: "Scheduled: (number)" but it doesn't craft. When i press cancel i can use the crafted parts to craft the machine.

I'm using the latest ATM3 and Java Versions

**Comment by notacompletemoron** [comment]
This is an ongoing issue with AE2, and other mods (including Mekanism).

There are just certain mods that it refuses to autocraft.

**Comment by Yoosk** [comment]
Edit your pattern to allow ore-dict substitutions, worked out for me. AE has issues with mek tanks, tablets etc, and sees each one differently.

**Comment by notacompletemoron** [comment]
@Yoosk it's an issue with how AE2/RS handle NBT/metadata in the crafting patterns.

https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/2346

**Comment by Yoosk** [comment]
Correct, but it works for me.

**Comment by Sakata-MC** [comment]
This has been an issue with mekanism and a few other mods (ic2 is one) for a long long time. I used to set up massive networks with LogisticsPipes, and had the exact same issue.  It has to do with how Mek/IC2 did they're own recipes. If you select the recipe form the gui and add it to crafting table/pattern, normally the item(s) with energy have different data than the one you get when you craft it yourself.

My workaround was to create pattern for tier1 item, then request it, put in the pattern for the t2 item, then replace the recipe with the t1 item I ordered, and then request the t2 item and make recipe for t3, etc.

LP and AE did eventually handle it better but it was always finicky.  The  above method works though.
